src/dbus_fast/message_bus.py

```diff
@@ -27,13 +27,34 @@
 
 MESSAGE_TYPE_CALL = MessageType.METHOD_CALL
 MESSAGE_TYPE_SIGNAL = MessageType.SIGNAL
-NO_REPLY_EXPECTED_VALUE = MessageFlag.NO_REPLY_EXPECTED.value
+NO_REPLY_EXPECTED = MessageFlag.NO_REPLY_EXPECTED
+_LOGGER = logging.getLogger(__name__)
+
 
 _Message = Message
 
 
 def _expects_reply(msg: _Message) -> bool:
-    return not (msg.flags.value & NO_REPLY_EXPECTED_VALUE)
+    """Whether a message expects a reply."""
+    return not (msg.flags & NO_REPLY_EXPECTED)
```

Comment:
This turned out to be faster with pure python by slower with cython. I'm going to revert it

Reply:
https://github.com/Bluetooth-Devices/dbus-fast/releases/tag/v1.91.1